.changeset/wise-teeth-study.md

```diff
@@ -0,0 +1,20 @@
+---
+'@backstage/plugin-catalog-backend-module-azure': minor
```

Comment:
```suggestion
'@backstage/plugin-catalog-backend-module-azure': patch
```

As the existing functionality still work this can be set as just a `patch`